JT/10_oop_basics/1 classes/class examples/3 abstract class/what_is_an_abstrcat_class.adoc

```diff
@@ -0,0 +1,69 @@
+= Abstract class
+:toc:
+:icons: font
+:url-quickref: https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/
+
+An *abstract class* is a class that cannot be initiated by itself, it needs to be sub-classed by another class to use its properties.
+
```

Comment:
Also could be added:
It can have constructors and static methods also.
It can have final methods which will force the subclass not to change the body of the method.

Reply:
Done